tests/integrationtest/r/expression/uuid.result

```diff
@@ -45,3 +87,28 @@ CREATE TABLE t1234 (id int PRIMARY KEY, uuid BINARY(16) DEFAULT (UUID_TO_BIN(UUI
 CREATE TABLE t12345 (id int PRIMARY KEY, uuid BINARY(16) DEFAULT (UUID_TO_BIN(UUID_V4())));
 CREATE TABLE t12346 (id int PRIMARY KEY, uuid BINARY(16) DEFAULT (UUID_TO_BIN(UUID_V7())));
 DROP TABLE IF EXISTS t123, t1234, t12345, t12346;
+CREATE TABLE u1 (
+uuid BINARY(16) PRIMARY KEY
+);
+INSERT INTO u1 VALUES
+(UUID_TO_BIN("5f13f854-d74a-11f0-9b7a-0ae0156bd76b")),
+(UUID_TO_BIN("c6437ef1-5b86-3a4e-a071-c2d4ad414e65")),
+(UUID_TO_BIN("a3e3b4a1-ea6d-471e-9860-8303a8b261f6")),
+(UUID_TO_BIN("271a8175-dadd-5df9-b0bd-20a4a0b441e6")),
+(UUID_TO_BIN("1f0e48c1-7860-69cc-9b3f-35f89c103d4d")),
+(UUID_TO_BIN("019b1440-87b7-7380-ab00-ce413e795004"));
+SELECT
+BIN_TO_UUID(uuid),
+UUID_VERSION(BIN_TO_UUID(uuid)),
+UUID_TIMESTAMP(BIN_TO_UUID(uuid)),
+FROM_UNIXTIME(UUID_TIMESTAMP(BIN_TO_UUID(uuid)))
+FROM u1
+ORDER BY uuid;
+BIN_TO_UUID(uuid)	UUID_VERSION(BIN_TO_UUID(uuid))	UUID_TIMESTAMP(BIN_TO_UUID(uuid))	FROM_UNIXTIME(UUID_TIMESTAMP(BIN_TO_UUID(uuid)))
+019b1440-87b7-7380-ab00-ce413e795004	7	1765571332.023000	2025-12-13 04:28:52.023000
+1f0e48c1-7860-69cc-9b3f-35f89c103d4d	6	1766995078.970004	2025-12-29 15:57:58.970004
+271a8175-dadd-5df9-b0bd-20a4a0b441e6	5	0	1970-01-01 08:00:00.000000
+5f13f854-d74a-11f0-9b7a-0ae0156bd76b	1	1765537487.118139	2025-12-12 19:04:47.118139
+a3e3b4a1-ea6d-471e-9860-8303a8b261f6	4	0	1970-01-01 08:00:00.000000
+c6437ef1-5b86-3a4e-a071-c2d4ad414e65	3	0	1970-01-01 08:00:00.000000
```

Comment:
I assume this test would fail by `NULL` agains `0` for these three lines, right?

Reply:
I've updated the result